examples/src/main/java/io/grpc/examples/dualstack/DualStackServer.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2024 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.examples.dualstack;
+
+import io.grpc.Server;
+import io.grpc.ServerBuilder;
+import io.grpc.examples.helloworld.GreeterGrpc;
+import io.grpc.examples.helloworld.HelloReply;
+import io.grpc.examples.helloworld.HelloRequest;
+import io.grpc.examples.loadbalance.LoadBalanceServer;
+import io.grpc.netty.shaded.io.grpc.netty.NettyServerBuilder;
```

Comment:
I don't really want users referencing the shaded name. I feel like it is internal. (I know users are doing it, and at times it is painful to avoid, but still.) We also don't want to change the dependencies for helloworld for this example, so I think we should create a new gradle project for this example and have it depend on grpc-netty instead of grpc-netty-shaded.